libraries/abstractions/cellular/test/test_cellular_api.c

```diff
@@ -177,6 +177,10 @@
 
 #define MESSAGE_BUFFER_LENGTH                ( 256U )
 
+/* RAT priority count for testing. This value should larger or equal to
+ * CELLULAR_MAX_RAT_PRIORITY_COUNT. */
+#define TEST_MAX_RAT_PRIORITY_COUNT          ( 3U )
```

Comment:
according to comments, maybe we can add 
#if CELLULAR_MAX_RAT_PRIORITY_COUNT > TEST_MAX_RAT_PRIORITY_COUNT
    #error "TEST_MAX_RAT_PRIORITY_COUNT  should not larger or equal to CELLULAR_MAX_RAT_PRIORITY_COUNT "
#endif

Reply:
It is a good suggestion. This will be updated.